source/deployment/microsoft-integrations.rst

```diff
@@ -0,0 +1,37 @@
+Microsoft Tools
+============================================
+
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect Microsoft tools in Mattermost. Here are some popular options:
+
+Office 365 Calendar (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb' when in a meeting scheduled via Outlook Calendar.
+- Supports <XXX?>
```

Comment:
Should we remove this or is it a placeholder?

Reply:
Good catch, meant as a question/placeholder for @aaronrothschild

Reply:
LGTM